app/scripts/views/EventsShow.js

```diff
@@ -64,6 +64,18 @@ class EventsShow extends Component {
     )
   }
 
+  renderTicketUrl() {
+    return <p>{ this.props.resourceData.ticketUrl }</p>
+  }
+
+  renderWebsiteUrl() {
+    return <p>{ this.props.resourceData.websiteUrl }</p>
+  }
+
+  renderSocialMedia() {
+    return <p>{ this.props.resourceData.socialMedia }</p>
```

Comment:
What do you imagine being presented in that field? Right now it looks like I could write anything in it ..

How about naming it `additionalInfo` or something and making it an markdown field?

Reply:
I thought it would be a list of social media handles. I looked into doing something fancy, like having an array of inputs, but it might be too much work for me right now. So then I thought about just having a more instructive description. Like: "List any relevant social media handles:"

Reply:
They wouldn't be clickable tho, an markdown rendered field could help to at least create links. And as its then inviting to use it for "everything else" we could also just make it an "everything else" field maybe?

Reply:
yeh, good call!

Reply:
https://github.com/adzialocha/hoffnung3000/pull/92/commits/29cedad82e1f15e64793dbf0f47ffeb4a0032049